stdlib/Statistics/test/runtests.jl

```diff
@@ -518,6 +518,9 @@ end
             @inferred quantile(T[1, 2, 3], (S(0.5), S(0.6)))
         end
     end
+    x = [3; 2; 1]
+    y = zeros(3)
+    @test quantile!(y, x, [0.1, 0.5, 0.9]) === y
```

Comment:
AFAICT, this tests the "in-placeness" of `quantile!`. I'd suggest to also add the other test from the docstring:
```julia
@test y == [1.2, 2.0, 2.8]
```